qemu/tests/cfg/live_backup.cfg

```diff
@@ -0,0 +1,36 @@
+- live_backup:
+    virt_test_type = qemu
+    type = live_backup
+    no vmdk qed
+    only Linux
```

Comment:
yes, it really not related to guest os. but we arrange our test plan by guest sometimes, if your set it to only Linux this feature will be skip to test if the test round is use windows guest.  so you can't set it to only Linux here.
For Md5sum check fail on windows, I guess you need to upgrade md5sum application in winutils.

Thanks,
Xu